app/models/manageiq/providers/azure/inventory/parser/cloud_manager.rb

```diff
@@ -449,7 +449,8 @@ def resource_status_message(resource)
 
   def build_image_name(image)
     # Strip the .vhd and Azure GUID extension, but retain path and base name.
-    File.join(File.dirname(image.name), File.basename(File.basename(image.name, '.*'), '.*'))
+    path = File.join(File.dirname(image.name), File.basename(File.basename(image.name, '.*'), '.*'))
```

Comment:
can you use `build_image_name` method in `refresh_helper_methods`?

Reply:
I'd have to include the module. I don't think it will be an issue, but I'll give it a try.